src/components/Capture/CountrySelector/index.tsx

```diff
@@ -0,0 +1,92 @@
+import { h, FunctionComponent } from 'preact'
+import { useState } from 'preact/compat'
+import { omit } from 'lodash/fp'
```

Comment:
We have an omit function in utils. Can we use that one? How much is this adding in terms of bundle size